anti_debug.cpp

```diff
@@ -662,7 +662,7 @@ int security::internal::virtualization::vm() {
 	mbstowcs(w_provider, s_provider, strlen(s_provider) + 1);
 
 	h_key = 0;
-	char* s_subkey = xor("SYSTEM\\CurrentControlSet\\Enum\\IDE");
+	const char* s_subkey = xor("SYSTEM\\CurrentControlSet\\Enum\\IDE");
```

Comment:
`xor` might return a pointer to const data, this change allows the code to compile in this case.